/*
 *     Copyright 2026 Parresum Soft @ http://parresum.de
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *          http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package de.parresum.kicad.parser.project;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 *
 *
 * @author Kai Uwe Bachmann
 */
public enum Severity {
   @JsonProperty("ignore")
   Ignore,

   @JsonProperty("warning")
   WARNING,

   @JsonProperty("error")
   ERROR;
}
